CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
+## 2.12.10 (2023-02-16)
+
+This release updates the bundled Boost and OpenSSL on Windows (#9686)
```

Comment:
Should mentions the versions. One important reason for this release are security fixes in OpenSSL, so I'd say the new version is relevant here.